openmetadata-ui/src/main/resources/ui/cypress/common/common.js

```diff
@@ -491,6 +492,15 @@ export const visitEntityDetailsPage = (
   interceptURL('GET', `/api/v1/search/suggest?q=*&index=*`, 'searchQuery');
   interceptURL('GET', `/api/v1/search/*`, 'explorePageSearch');
   const id = dataTestId ?? `${serviceName}-${term}`;
+
+  if (entityType) {
+    cy.get('[data-testid="global-search-selector"]').click();
+    cy.get('.global-search-select-menu')
+      .should('be.visible')
+      .get(`[data-testid="global-search-select-option-${entityType}"]`)
+      .click();
```

Comment:
is it intentional to use .get after .should?, if you want search within ".global-search-select-menu", you can chain .find, also for selector use data testid

Reply:
I'll see if we can add data testid to the antd select